client/src/modules/fee_center/modals/fee_center.modal.html

```diff
@@ -0,0 +1,176 @@
+            ng-value="1"
+            ng-model="FeeCenterModalCtrl.feeCenter.is_principal"
+            ng-click="FeeCenterModalCtrl.auxiliaryFee(false)"
+            id="principal"
+            >
+          <span translate>FORM.LABELS.PRINCIPAL_2</span>
+        </label>
+
+        <label class="radio-inline">
+          <input
+            type="radio"
+            name="is_principal"
+            ng-value="0"
+            ng-model="FeeCenterModalCtrl.feeCenter.is_principal"
+            ng-click="FeeCenterModalCtrl.auxiliaryFee(true)"
+            id="auxiliary"
+            >
+          <span translate>FORM.LABELS.AUXILIARY</span>
+        </label>
+      </div>
+    </div>
+    <hr />
+
+    <div style="padding-left: 5%" class="form-group">
+      <div ng-if="FeeCenterModalCtrl.auxiliaryCenter" class="form-group">
+        <div class="radio">
+          <label class="radio-inline">
+            <input
+              type="radio"
+              name="auxiliary"
+              ng-value="1"
+              ng-model="FeeCenterModalCtrl.feeCenter.is_cost"
+              ng-click="FeeCenterModalCtrl.costCenter(true)"
+              id="is_cost"
+              >
+            <span translate>FORM.LABELS.COST_CENTER </span>
+          </label>
+
+          <label class="radio-inline">
+            <input
+              type="radio"
+              name="auxiliary"
+              ng-value="0"
+              ng-model="FeeCenterModalCtrl.feeCenter.is_cost"
+              ng-click="FeeCenterModalCtrl.costCenter(false)"
+              id="is_profit"
+              >
+            <span translate>FORM.LABELS.PROFIT_CENTER </span>
+          </label>
+        </div>
+      </div>  
+
+      <div ng-if="FeeCenterModalCtrl.feeCenter.is_principal" class="checkbox">
+        <label>
+          <input
+            type="checkbox"
+            id="has_profit_center"
+            ng-true-value="1"
+            ng-false-value="0"
+            ng-model="FeeCenterModalCtrl.hasProfitCenter">
+          <span translate>
+            <strong>
+              FORM.INFO.ASSIGNED_PROFIT_CENTER
+            </strong>
+          </span>
+        </label>
+      </div>
+      <div ng-if="(FeeCenterModalCtrl.feeCenter.is_principal && FeeCenterModalCtrl.hasProfitCenter) || (FeeCenterModalCtrl.auxiliaryCenter && FeeCenterModalCtrl.isProfitCenter)">
+        <bh-account-reference-select
+          id="account_profit_reference_id"
+          account-reference-id="FeeCenterModalCtrl.feeCenter.reference_profit_id"
+          label="FORM.LABELS.PROFIT_CENTER"
+          on-select-callback="FeeCenterModalCtrl.onSelectAccountReference(accountReference, 0)"
+          required="true"
+          validation-trigger="FeeCenterForm.$submitted">
+        </bh-account-config-select>
+      </div>
+      
+      <span ng-if="FeeCenterModalCtrl.feeCenter.is_principal">
+        <hr />
+      </span>
+
+      <div ng-if="FeeCenterModalCtrl.feeCenter.is_principal" class="checkbox">
```

Comment:
It's just because I use the same <bh-account-reference-select> as it is for the case of the Principal Fee Center or the auxiliary centers (Costs or Profits)